src/main/java/com/redhat/devtools/lsp4ij/features/foldingRange/LSPFoldingRangeBuilder.java

```diff
@@ -44,6 +48,14 @@ public class LSPFoldingRangeBuilder extends CustomFoldingBuilder {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LSPFoldingRangeBuilder.class);
 
+    // TODO: Can this be derived from the language server? If not, these are the standard for modern languages
+    private static final Map<Character, Character> BRACE_PAIR_CHARS_FWD = Map.of(

```

Comment:
Are these available via the language server? We could also make this part of client config, but my guess is that's going to be overkill. The vast majority of languages use paired braces/brackets/parens for block scope, and those that don't generally don't use any kind of block scope delimiters, e.g., Python. Thoughts?

Reply:
As discussed in Slack, JetBrains has long had a concrete assumption that structural block delimiters are basically limited to braces, brackets, and parens as evidenced in their own `AbstractFileType`:

![image](https://github.com/user-attachments/assets/42a4e155-0812-4bb6-9f5f-c9cb5eec3b4e)

In that same discussion, we decided that it should be fine to support exactly those same delimiters until/unless we learn of a language being integrated via a language server that supports something other than those for explicit block designation (i.e., not like Python or YAML where blocks are based on indentation).